wp/sections/feature.tex

```diff
@@ -3,18 +3,27 @@
 \subsection{Configuration file}
 \par
 It's worth starting with the configuration file. This is a file in which the user can manually turn rules on and off or configure the rules settings. Below is one of the rules in the configuration file.
-
```

Comment:
Is it okay that there are changes in many tex files? Are they part of this change?

Reply:
This is because of merge or because we made minor changes to WP and I don't want to create a new PR for this.